### ISSUE TITLE
Can't get the results reported in the paper

### ISSUE THREAD

**Issue by Wi-sc** [yinyunie/Total3DUnderstanding#23]
Hi,

I also have some problems about the training. I cannot get the results reported in the paper. I also tried to use released models to initialize the framework before joint training, but the results decrease. Can you help me check the config file? By the way, you mentioned blend Pixl3D and SUN RGBD datasets in the joint training, how to do it?

My training results:
layout_iou: 0.456693
iou_3d: 0.118330
iou_2d: 0.599362

This is my config setting:

method: TOTAL3D
resume: False
finetune: True
weight: ['out/pretrained_models/pretrained_model.pth', 'out/pretrained_models/meshnet_model.pth']
seed: 123
device:
use_gpu: True
num_workers: 2
data:
dataset: sunrgbd
split: data/sunrgbd/splits
tmn_subnetworks: 2
face_samples: 1
with_edge_classifier: True
model:
layout_estimation:
method: PoseNet
loss: PoseLoss
object_detection:
method: Bdb3DNet
loss: DetLoss
mesh_reconstruction:
method: DensTMNet
loss: ReconLoss
optimizer:
method: Adam
lr: 1e-4
betas: [0.9, 0.999]
eps: 1e-08
weight_decay: 1e-04
scheduler:
patience: 5
factor: 0.5
threshold: 0.01
train:
epochs: 200
phase: 'joint' # 'layout_estimation' or 'object_detection' or 'joint'.
freeze: ['mesh_reconstruction']
batch_size: 3
test:
phase: 'joint'
batch_size: 3
demo:
phase: 'joint'
log:
vis_path: visualization
save_results: True
vis_step: 100
print_step: 50
path: out/total3d
